---
layout: default
title: Samtools - Documentation
---

# Documentation
Documentation for SAMTools is available by using `man [tool name]` on the
command line. This reference material is included below for easy reference.

## Man Pages

* [Samtools]({{ site.baseurl }}man/samtools)
* [BCFtools]({{ site.baseurl }}man/bcftools)
* [tabix &amp; bgzip]({{ site.baseurl }}man/tabix)
* [faidx]({{ site.baseurl }}man/faidx)

## Specifications

SAMtools conforms to the specifications produced by the GA4GH File Formats working group. Details of the current specifications are available on the  [hts-specs page](http://samtools.github.io/hts-specs).

## Publications

* Li H.*, Handsaker B.*, Wysoker A., Fennell T., Ruan J., Homer N., Marth G., Abecasis G., Durbin R. and 1000 Genome Project Data Processing Subgroup (2009) The Sequence alignment/map (SAM) format and SAMtools. Bioinformatics, 25, 2078-9. _[PMID: [19505943](href="http://www.ncbi.nlm.nih.gov/pubmed/19505943)]_
* Li H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics. 2011 Nov 1;27(21):2987-93. Epub 2011 Sep 8. _[PMID: [21903627](http://www.ncbi.nlm.nih.gov/pubmed/21903627)]_
* Danecek P., Schiffels S., Durbin R. Multiallelic calling model in bcftools (-m) [http://samtools.github.io/bcftools/call-m.pdf]
* Li H. Improving SNP discovery by base alignment quality. Bioinformatics. 2011 Apr 15;27(8):1157-8. doi: 10.1093/bioinformatics/btr076. Epub 2011 Feb 13. _[PMID: [21320865](http://www.ncbi.nlm.nih.gov/pubmed/21320865)]_
* Durbin R. Segregation based metric for variant call QC _[[link](http://samtools.github.io/bcftools/rd-SegBias.pdf)]_
* Li H, Mathematical Notes on SAMtools Algorithms _[[link](http://www.broadinstitute.org/gatk/media/docs/Samtools.pdf)]_

